python/vTestInterface.py

```diff
@@ -140,8 +140,13 @@ def generateTestInfo(test):
 
 def getTestDataVCAST(enviroPath):
 
-    api = UnitTestApi(enviroPath)
-
+    # dataAPI throws if there is a tool/enviro missmatch
+    try:
+        api = UnitTestApi(enviroPath)
+    except Exception as err:
```

Comment:
What does it actually throw?

Reply:
File "vector/apps/DataAPI/migrations/migrate.py", line 44, in perform_necessary_migrations
```
raise MigrationError('''Master.db (version %i) is newer than your version of VectorCAST (%i).

Reply:
Won't fix.